### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by hungtranpro2**

### CODE REVIEW

**Comment by hungtranpro2** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MovieTicketBooking</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (chienpv-0644) [inline app/views/layouts/application.html.erb]: nháy ""

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MovieTicketBooking</title>
```
Comment (chienpv-0644) [inline app/views/layouts/application.html.erb]: I18n

**Comment by chienpv-0644** [comment]
commented

**Comment by hungtranpro2** [comment]
ready

**Pull request merged**